partner_contact_company/__manifest__.py

```diff
@@ -4,27 +4,19 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Partner Company",
-    "summary": "Partner Company",
-    "version": "8.0.1.0.0",
-    "category": "Partner",
-    "website": "www.akretion.com",
-    "author": " Akretion",
-    "license": "AGPL-3",
-    "application": False,
-    'installable': False,
-    "external_dependencies": {
-        "python": [],
-        "bin": [],
-    },
-    "depends": [
-        "base",
-    ],
-    "data": [
-        "views/partner_view.xml",
+    'name': 'Partner Company',
+    'version': '10.0.1.0.0',
+    'author': "Akretion, "
+              "Serpent Consulting Services Pvt. Ltd.",
```

Comment:
I should add this in the Readme of the project (to have some guide line and avoid frustration), but I think that migrating a module is not enough to be set as an author of the module. Some special case can deserve this promotion (like huge refactoring). But for a simple migration I think is not enough. Thanks for understanding my point of view.